### PR TITLE
add becker integration doc

### PR DESCRIPTION
**Pull request by nicolasberthel**
## Proposed change

Documentation for new integration for becker covers

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35288
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/becker.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Cover
+
+## Configuration
+
+The becker cover configuration consist in a new becker platform in your cover configuration:
+
+```yaml
+# Example configuration.yaml entry
+cover:
+  - platform: becker
```
Comment (frenck) [inline source/_integrations/becker.markdown]: New integrations configured via a platform is no longer allowed.
Please adjust the integration. 👍

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/becker.markdown (hunk 1 above)
Comment (nicolasberthel) [reply source/_integrations/becker.markdown]: Integration changed

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by cgarwood** [comment]
PR seems to have gone stale. Feel free to submit a new PR when you're ready to finish 👍

**Pull request closed without merging**